src/commcare_cloud/environment/main.py

```diff
@@ -123,8 +123,11 @@ def users_config(self):
 
     @memoized_property
     def terraform_config(self):
-        with open(self.paths.terraform_yml) as f:
-            config_yml = yaml.load(f)
+        try:
+            with open(self.paths.terraform_yml) as f:
+                config_yml = yaml.load(f)
+        except IOError:
+            return None
```

Comment:
Does the thing that calls this check if the result is `None`?

In general, I'm not really a fan of this pattern of error handling since it forces the caller to inspect the returned object before using it. The alternatives all probably involve more work though, so I guess it's reasonable to do it this way.

Reply:
Yeah the annoying thing here is that this needs to be pickleable (since this object needs to be picklable). The calling code doesn't check this currently, so on an env that doesn't use terraform (all of them but the test one currently) this will give a traceback to this point if you run `cchq <env> terraform ...`. Not the greatest, but for the level of maturity we're currently at with this feature I think it's roughly appropriate.